jobs/ml-ops/registry/README.md

```diff
@@ -0,0 +1,29 @@
+# Scaleway private registry for docker images
```

Comment:
I'm not sure if it's worth splitting the Terraform config like this, you can have a Terraform config that deploys everything. Including creating the registry namespace and pushing the images.

You can take inspiration from this: https://github.com/cyclimse/severless-pypi/blob/main/deploying/terraform/images.tf